app/views/course/assessment/question/text_responses/_text_response.json.jbuilder

```diff
@@ -1,13 +1,16 @@
 # frozen_string_literal: true
 json.autogradable question.auto_gradable?
+default_max_attachment_size_mb = Course::Assessment::Question::TextResponse::DEFAULT_MAX_ATTACHMENT_SIZE_MB
 
 case question.question_type_sym
 when :file_upload
   json.maxAttachments question.max_attachments
+  json.maxAttachmentSize question.max_attachment_size || default_max_attachment_size_mb
```

Comment:
I think it's better to define the logic inside the model. You can create a method that contains this business logic (together with line 13) and return a number. This way, we remove business-level logic from view component.

Reply:
If the logic is abstracted away, we can always reuse the method (ie in text_responses/edit.json.jbuilder). This prevents repeatable code. 

If you write it once it's fine. if you write it twice (maybe) it's fine. If you repeat thrice, then abstract it away.

Reply:
done